client/components/data/document-head/index.jsx

```diff
@@ -81,6 +81,7 @@ class DocumentHead extends Component {
 
 DocumentHead.propTypes = {
 	title: TranslatableString,
+	formattedTitle: TranslatableString,
```

Comment:
I think it may be worth adding this to the existing list of props within the documentation: https://github.com/Automattic/wp-calypso/blob/master/client/components/data/document-head/README.md

Reply:
Good catch, I missed that! Done in https://github.com/Automattic/wp-calypso/pull/41803/commits/5751f65a74b718e52a22619b646bd2dedc91fdb9